operational/fnms/schedule_fnms_reports/README.md

```diff
@@ -52,6 +52,12 @@ This policy has the following input parameters required when launching the polic
 ### Policy Actions
 No actions
 
+### Required RightScale Roles
+ 
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```

Comment:
for consistency please change to Cloud Management - credential_viewer

Reply:
Changed to Cloud Management - credential_viewer